src/Faker/Provider/Person.php

```diff
@@ -73,6 +73,22 @@ public function firstName($gender = null)
         return $this->generator->parse(static::randomElement(static::$firstNameFormat));
     }
 
+    public function profilePicture($gender = null)
```

Comment:
no, please make it a standalone package. I don't want to add real images to the Faker package, hich is heavy enough without that.